index.html

```diff
@@ -400,6 +399,29 @@
 		<script src="bower_components/jquery/dist/jquery.min.js"></script>
 		<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
 		<script src="site-assets/main.js"></script>
+		<script>
+			(function () {
+				var webComponentsSupported = (
+					'registerElement' in document &&
+					'import' in document.createElement('link') &&
+					'content' in document.createElement('template')
+				);
+
+				if (!webComponentsSupported) {
+					var script = document.createElement('script');
+					script.async = true;
+					script.src = 'bower_components/webcomponentsjs/webcomponents-lite.min.js';
+					script.onload = finishLazyLoading;
+					document.head.appendChild(script);
+				} else {
+					finishLazyLoading();
+				}
+
+				function finishLazyLoading() {
+				  window.Polymer = window.Polymer || {dom: 'shadow'};
```

Comment:
That example doesnt have an onload handler that settings native shadow dom. This needs to be done before polymer loads. It's safest to do outside of this dance high in the page before all imports.